src/SFA.DAS.FAT.Web/Models/CoursesViewModel.cs

```diff
@@ -1,22 +1,61 @@
-﻿using System.Collections.Generic;
+﻿using System;
+using System.Collections.Generic;
 using System.Linq;
-using SFA.DAS.FAT.Domain.Courses;
 
 namespace SFA.DAS.FAT.Web.Models
 {
     public class CoursesViewModel
     {
         public List<CourseViewModel> Courses { get; set; }
         public string Keyword { get; set; }
+        public int Total { get ; set ; }
+        public int TotalFiltered { get ; set ; }
+        public List<SectorViewModel> Sectors { get ; set ; }
+        public string TotalMessage => GetTotalMessage();
+        public List<Guid> SelectedSectors { get ; set ; }
+        public Dictionary<string, string> ClearSectorLinks => BuildClearSelectedFilterLink();
+        public string ClearKeywordLink => BuildClearKeywordFilterLink();
+        
         private string GetTotalMessage()
         {
             var totalToUse = string.IsNullOrEmpty(Keyword) ? Total : TotalFiltered;
```

Comment:
this is different now that sectors might have reduced the total right?

Reply:
Yeah missed that one - added test and updated.